core/utils_test.py

```diff
@@ -63,19 +63,24 @@ def test_to_ascii(self) -> None:
 
     def test_yaml_dict_conversion(self) -> None:
         """Test yaml_from_dict and dict_from_yaml methods."""
-        test_dicts = [{}, {'a': 'b'}, {'a': 2}, {'a': ['b', 2, {'c': 3.5}]}]
+        test_dicts: List[Dict[str, Any]] = [
+            {},
+            {'a': 'b'},
+            {'a': 2},
+            {'a': ['b', 2, {'c': 3.5}]}
+        ]
 
         for adict in test_dicts:
-            yaml_str = python_utils.yaml_from_dict(adict) # type: ignore[no-untyped-call]
+            yaml_str = utils.yaml_from_dict(adict)
 
             yaml_dict = utils.dict_from_yaml(yaml_str)
             self.assertEqual(adict, yaml_dict)
 
-        with self.assertRaisesRegexp( # type: ignore[no-untyped-call]
+        with self.assertRaisesRegexp(  # type: ignore[no-untyped-call]
             utils.InvalidInputException,
             'while parsing a flow node\n'
             'expected the node content, but found \'<stream end>\'\n'):
-            yaml_str = utils.dict_from_yaml('{')
+            yaml_str = utils.dict_from_yaml('{')  # type: ignore
```

Comment:
Can you try to remove the ignore and push your code so that I can try to debug the error?

Reply:
okay removed , it shows error core/utils_test.py:83: error: Incompatible types in assignment (expression has type "Dict[str, Any]", variable has type "str") [assignment]
Found 1 error in 1 file (checked 1 source file)

Reply:
You will need to use a different variable name

Reply:
@DiptoChakrabarty now its complaining that variable is unused so you can carry out that test without storing it in a varible